CHANGELOG.rst

```diff
@@ -5,6 +5,21 @@ Changelog
 This document records all notable changes to `HTTPie <https://httpie.org>`_.
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.7.0 (2022-02-07)
+------------------
+
+**Added**
+
+* Update github workflows
+* Update License
+* Remove reference to parent pom
```

Comment:
You did not remove the parent pom 😅  At least I cant see it in the pom.. and then you also dont need to increase the version

Reply:
just removed a comment that referenced the parent pom, but you are correct about the version!